Update install and setup documentation and add in quick demo (#10)
ros-industrial/rmf_scheduler: README.md

[![support level: consortium / vendor](https://img.shields.io/badge/support%20level-consortium-brightgreen.svg)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)

> Manages task schedules for OpenRMF.

Reviewer: ```suggestion
Manages task schedules for Open-RMF.
```

## Requirements

* ROS 2 Humble

> * OpenRMF (More information [here](https://github.com/open-rmf/rmf))

Reviewer: ```suggestion
* Open-RMF (More information [here](https://github.com/open-rmf/rmf))
```

## Documentation

Refer to the [documentation](https://rmf-scheduler.readthedocs.io/) for usage instructions.

## Quick Setup

> Full setup instructions with OpenRMF can be found in the documentation above.  

Reviewer: ```suggestion
Full setup instructions with Open-RMF can be found in the documentation above.  
```
